cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -170,7 +170,7 @@ jobs:
           gh release upload ${{ github.ref_name }} $(cat uploads.txt)
           echo "uploaded!"
 
-{{%- if tap %}}
+{{%- if 'homebrew' in publish_jobs and tap %}}
```

Comment:
Somewhere in the rust code should probably error out if the first part is true but the second part isn't. The "npm has to have tar.gz" checks might be a good example to go off of..?

Reply:
also ignorable if we just want it out the door

Reply:
We don't have an error case for this in npm right now except in `init`, and the Homebrew `init` prevents the case from existing. I did add a warning for if it comes up during rendering though, same as with npm.